### ISSUE TITLE
MessageChat not exported

### ISSUE THREAD

**Issue by shouya** [telegram-rs/telegram-bot#85]
I'm playing with this library (master) but find that it's unable to pattern matching on `chat` field of a `Message`.

The `Message` struct looks like:

```rust
pub struct Message {
    pub id: MessageId,
    pub from: User,
    pub date: Integer,
    pub chat: MessageChat,
    pub forward: Option<Forward>,
    pub reply_to_message: Option<Box<MessageOrChannelPost>>,
    pub edit_date: Option<Integer>,
    pub kind: MessageKind,
}
```

The `chat` field has type of `MessageChat`, which is defined in `telegram_bot_raw::types::chat` module. This module is not re-exported in the top level module of telegram-bot crate, so I'm not able to use it directly.

Now I have to include the telegram-bot-raw crate to achieve my goal, but I think this might not be intended.

**Comment by knsd** [comment]
Oh, you are right, thanks for notice. I forgot to re-export some types: `MessageChat`, `MessageOrChannelPost` and `ChannelPost`

I will update upload new version tomorrow.

**Comment by knsd** [comment]
Fixed in 0.6.1.
